src/environment/environment.ts

```diff
@@ -22,5 +22,8 @@ export const environment = {
         port: parseInt(process.env.POSTGRES_PORT ?? "5432"),
     },
     appPort: parseInt(process.env.PORT ?? "8080"),
-    authPermissionUrl: process.env.AUTH_PERMISSION_URL
+    authPermissionUrl: process.env.AUTH_HOST+'/api/v1/hasPermission',
+    serviceUrl: process.env.SERVICE_URL,
```

Comment:
serviceUrl not required

Reply:
Missing package.json command to run unit tests

Reply:
enable disable mock changes what we did for flex is missing

Reply:
removed serviceUrl

updated commands in package.json to run unit and integration tests

Reply:
added test_env condition for enable disable mock changes